neuro-cli/src/neuro_cli/disks.py

```diff
@@ -165,17 +187,23 @@ async def create(
     "disk", type=UnionType("disk", PlatformURIType(allowed_schemes=("disk",)), DISK)
 )
 @option("--full-uri", is_flag=True, help="Output full disk URI.")
-async def get(root: Root, cluster: Optional[str], disk: str, full_uri: bool) -> None:
+async def get(
+    root: Root, cluster: Optional[str], disk: Union[str, URL], full_uri: bool
+) -> None:
     """
     Get disk DISK_ID.
     """
-    disk_id = await resolve_disk(disk, client=root.client, cluster_name=cluster)
-    disk_obj = await root.client.disks.get(disk_id, cluster_name=cluster)
+    if isinstance(disk, URL):
+        disk_str = disk.parts[-1]
+    else:
+        disk_str = disk
```

Comment:
could you please explain why we dropped `resolve_disk` here?

Reply:
Good question, originally the intent was to simplify this logic. When I started to explain possible cases, I find out that my reasoning was wrong and we still need this method for cases when we have disk URLs in full form or with the owner.
Reverted in [c381a1e](https://github.com/neuro-inc/neuro-cli/pull/2960/commits/c381a1ecf85a8bc5423310dac2449a8259a72415).